roseus/defun.h

```diff
@@ -0,0 +1 @@
+// redefine defun for update defun() API (https://github.com/euslisp/EusLisp/pull/116)
```

Comment:
defun.h is auto-generated in CMakeLists (https://github.com/jsk-ros-pkg/jsk_roseus/blob/master/roseus/CMakeLists.txt#L108).
We should keep it out of this PR (and potentially add it to gitignore)

Reply:
Thank you!